tests/plugins/clipboard/datatransfer.js

```diff
@@ -1193,5 +1193,32 @@ bender.test( {
 		assert.areSame( '', dt1._stripHtml( '' ), 'Empty html' );
 		assert.isUndefined( dt1._stripHtml( undefined ), 'Undefined' );
 		assert.isNull( dt1._stripHtml( null ), 'Null' );
+	},
+
+	// (#1832)
+	'test setData with empty string doesn\'t call nativeDataTransfer.setData': function() {
```

Comment:
This test is incorrectly written. To really check if `setData` is not called, it's enough to call `setData( type, '' )` for several MIME types (IMO it'd be enough even for only one). There is no need to set correct data.

After this change the assertion will be simplified to checking if `spy.callCount` equals `0`.